src/topology/metric_space/emetric_space.lean

```diff
@@ -390,13 +390,25 @@ def pseudo_emetric_space.induced {α β} (f : α → β)
   end }
 
 /-- Pseudoemetric space instance on subsets of pseudoemetric spaces -/
-instance {α : Type*} {p : α → Prop} [t : pseudo_emetric_space α] :
-  pseudo_emetric_space (subtype p) := t.induced coe
+instance {α : Type*} {p : α → Prop} [pseudo_emetric_space α] : pseudo_emetric_space (subtype p) :=
+pseudo_emetric_space.induced coe ‹_›
 
 /-- The extended psuedodistance on a subset of a pseudoemetric space is the restriction of
 the original pseudodistance, by definition -/
 theorem subtype.edist_eq {p : α → Prop} (x y : subtype p) : edist x y = edist (x : α) y := rfl
 
+namespace mul_opposite
+
+/-- Pseudoemetric space instance on multiplicative opposites of pseudoemetric spaces -/
+@[to_additive]
+instance {α : Type*} [pseudo_emetric_space α] : pseudo_emetric_space αᵐᵒᵖ :=
+pseudo_emetric_space.induced unop ‹_›
```

Comment:
It looks like you forgot about the `emetric_space` instance. BTW, is there a diamond for `metric_space.to_emetric_space`?

Reply:
I've no idea if there's a diamond, I hope whoever wrote `induced` thought of that. Thanks for pointing out `emetric_space`, I've added the instances.